contrib/widget-examples/lemonbar/title.lua

```diff
@@ -0,0 +1,11 @@
+dofile 'lib.lua'
+
+widget = {
+  plugin = 'xtitle',
+  cb = function(t)
+    t = t or ''
+    -- need utf8 lib
+    t = (#t < 100 and t) or string.sub(t, 1, 100) .. '...'
+    return '%{c}' .. colorize(t, '#ffe') .. '%{r}'
```

Comment:
Use `luastatus.barlib.escape` to prevent lemonbar from interpreting %-sequences in `t` :)